composer.json

```diff
@@ -29,7 +29,8 @@
         "aws/aws-sdk-php"              : "~2.4",
         "pda/pheanstalk"               : "~3.0",
         "league/container"             : "~1.0",
-        "videlalvaro/php-amqplib"      : "~2.5"
+        "videlalvaro/php-amqplib"      : "~2.5",
+        "phpunit/phpunit"              : "~4.5.0"
```

Comment:
why? `phpunit` is a development requirement, as you can run `./vendor/bin/phpunit`. In order to run the tests, I had to manually look up which version of phpunit was being used, and then download it myself (i'm using phpunit < 4)